lib/entity/contact/contact.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:equatable/equatable.dart';
+
+class Contact extends Equatable {
```

Comment:
IMO, should be separate Contact vs PresentationContact vs Contact from OS bla bla bla